### ISSUE TITLE
Missing configuration parameters cause NPEs

### ISSUE THREAD

**Issue by propensive** [scalacenter/bloop#203]
If a configuration file is missing an expected parameter, the outcome in at least certain cases is a null-pointer exception being thrown in the server.

Supporting hand-written configuration files probably shouldn't be your highest priority, but it should be a relatively easy fix.

**Comment by jvican** [comment]
> If a configuration file is missing an expected parameter, the outcome in at least certain cases is a null-pointer exception being thrown in the server.
  
Yeah, I'm aware of this. I'd like to solve this issue together with the refactoring to use metaconfig, which I'll do in the upcoming week.

**Comment by propensive** [comment]
When you do, please spare a thought for people wanting to interact with the configuration files through shell scripting. The current format is quite convenient as I'm able to work with it using tools like `grep`, `cut` and `tr`.

Most of the time, I'm not reading them; I'm writing them, so don't get too hung up about this concern of mine. If you've got other reasons to switch to a new format, they probably outweigh this minor issue. It's probably not good practice to be reading configuration files intended for Bloop, anyway... ;)
